packages/react/src/patterns/blocks/ContentBlockSegmented/ContentBlockSegmented.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { ContentBlock } from '../../sub-patterns/ContentBlock';
+import { ContentGroup } from '../../sub-patterns/ContentGroup';
+import { ContentItem } from '../../sub-patterns/ContentItem';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import { Image } from '../../../components/Image';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { settings } from 'carbon-components';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * Content Block - Simple pattern
+ *
+ * @param {object} props props object
+ * @param {string} props.heading content block heading
+ * @param {string} props.copy content block short copy to support the heading
+ * @param {object} props.items content block content items
+ * @returns {*} Content Block - Segmented pattern
+ */
+const ContentBlockSegmented = ({ copy, heading, items }) => {
```

Comment:
I think there should be an image prop here for the `ContentBlock`.

Reply:
The images are on a segment level though?